wk0mcq/mcq.js

```diff
@@ -179,7 +179,7 @@ const _12D = "target.push(newElement)";
 
 // YOUR CODE HERE
 // SOLUTION STARTS
-const yourChoice12 = FILL_ME_IN;
+const yourChoice12 = _12D;
```

Comment:
You can test this file by typing in npm test in your terminal line - don't think this passes